### PR TITLE
Switch absolute links to relative ones - follow-up of #19317

### PR DESCRIPTION
**Pull request by SphinxKnight**
#### Summary
Changes two links in order for them to be relative and not absolute (as in other docs)

#### Motivation
Consistency

#### Related issues
#19317 

#### Metadata
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by Josh-Cena — APPROVED** [review]

File: files/en-us/mdn/community/issues/index.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ As a rule, do:
 
 - Use [GitHub discussion](https://github.com/mdn/mdn-community/discussions) before filing an issue. This helps to keep issues focused and productive.
 - Ask questions using other mechanisms like [chat rooms](https://chat.mozilla.org/#/room/#mdn:mozilla.org) or [forums](<(https://discourse.mozilla.org/c/mdn/236)>) if you are not sure whether something is useful or have a simple question.
```
Comment (Josh-Cena) [inline files/en-us/mdn/community/issues/index.md:62]: Pretty sure this should be

```suggestion
- Ask questions using other mechanisms like [chat rooms](https://chat.mozilla.org/#/room/#mdn:mozilla.org) or [forums](https://discourse.mozilla.org/c/mdn/236) if you are not sure whether something is useful or have a simple question.
```

Comment (SphinxKnight) [reply files/en-us/mdn/community/issues/index.md:62]: This PR was not about this so I'm unsure to include this here :/

Comment (Josh-Cena) [reply files/en-us/mdn/community/issues/index.md:62]: Well, this is _actually_ a broken link, so makes sense to fix all links in one PR :-)

**Review by Rumyra — APPROVED** [review]

File: files/en-us/mdn/community/issues/index.md (hunk 1 above)
Comment (Rumyra) [reply files/en-us/mdn/community/issues/index.md]: Yes normally changes outside the scope of the pr shouldn't be included for simplicity, however as this is a minor change I'm going to merge 👍

**Pull request merged**